bin/rucio-admin

```diff
@@ -96,6 +96,31 @@ logger = logging.getLogger("user")
 tablefmt = 'psql'
 
 
+def unindent(text, indent=0):
+    lines = text.split("\n")
+    if len(lines) < 2:
+        return text
+    headline, lines = lines[0], lines[1:]
+    indent_length = 0
+    for line in lines:
+        if line.strip():
+            i = 0
+            while line[i] == ' ':
+                i += 1
+            indent_length = i
+            break
+    out_lines = [headline.strip()]
+    remove_indent = " " * indent_length
+    add_indent = " " * indent
+    for line in lines:
+        if line.startswith(remove_indent):
+            line = line[indent_length:]
+        else:
+            line = line.lstrip()
+        out_lines.append(add_indent + line)
+    return "\n".join(out_lines)
+
+
```

Comment:
Maybe I misunderstood your motivation behind this, but I believe the use of `textwrap.dedent()` to be superior.